ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/ClientOptions.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.api.client;
+
+import io.confluent.ksql.api.client.impl.ClientOptionsImpl;
+
+public interface ClientOptions {
+
+  ClientOptions setHost(String host);
+
+  ClientOptions setPort(int port);
+
+  ClientOptions setUseTls(boolean useTls);
+
+  ClientOptions setUseClientAuth(boolean useClientAuth);
+
+  ClientOptions setVerifyHost(boolean verifyHost);
```

Comment:
These additional TLS options were needed to get the tests working. In a follow-up PR they'll either be removed in favor of exposing Vert.x HttpClientOptions (pending discussion in https://github.com/confluentinc/ksql/pull/5200#discussion_r419190143) or I'll refactor all the TLS options into a separate interface in order to clean up this one.